portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,61 @@
+html {
+  font-size: 100%;
+}
+
 #content {
   margin-left: auto;
   margin-right: auto;
   width: 650px;
 }
 
 #greeting-container {
-  margin-top: 20px;
+  margin-top: 1.25em;
 }
+
+.image-thumbnails img {
+    border: 1em solid rgb(132, 179, 249);
+    max-width:100%;
+    max-height:100%;
```

Comment:
nit: Here and throughout this file, let's use 2 spaces/tabs for property indentation, rather than 4 spaces/tabs. Just so things are nice and consistent throughout the stylesheet :-)

FYI a "nit" is a prefix we use to indicate a "nitpick that we think would improve the code but isn't mandatory".

Reply:
Will do! The CSS file given as an example had the 2 spaces as spacing, but I used 4. Thanks for catching this.